db/migrate/20200417145121_create_validation_errors.rb

```diff
@@ -0,0 +1,10 @@
+class CreateValidationErrors < ActiveRecord::Migration[6.0]
+  def change
+    create_table :validation_errors do |t|
+      t.string :form_object
+      t.json :details
```

Comment:
I think this needed to be `jsonb` - https://stackoverflow.com/questions/39637370/difference-between-json-and-jsonb-in-postgres